PhysicsTools/PatUtils/python/tools/muonRecoMitigation.py

```diff
@@ -62,11 +62,13 @@ def muonRecoMitigation(process,
     
     # now cleaning ================================
     cleanedPFCandCollection=cleanCollName+postfix
+
     if runOnMiniAOD:
-        cleanedPFCandProducer = cms.EDProducer("CandPtrProjector", 
-                                               src = cms.InputTag(pfCandCollection),
-                                               veto = cms.InputTag(badMuonCollection)
-                                               )
+        import CommonTools.CandAlgos.candPtrProjector_cfi as _mod
+        cleanedPFCandProducer = _mod.candPtrProjector.clone(
+                                               src  = pfCandCollection,
+                                               veto = badMuonCollection
+                                         )
     else:
         cleanedPFCandProducer = cms.EDProducer("PFCandPtrProjector", 
```

Comment:
I am not able to find a PFCandPtrProjector plugin anywhere in CMSSW, not even in 9_1_X (which is the release where this config was introduced with #18165)

@mmarionncern and @zdemirag: do you remember where that plugin was defined?

By the way, isn't this config already obsolete (and candidate for possible deletion)? `corMETFromMuonAndEG` is only called by PhysicsTools/PatAlgos/python/slimming/extraSlimmedMETs_MuEGFixMoriond2017.py, which according to the name is quite likely not needed any more. (Moreover, `muCorrection=False` in there , and therefore this `muonRecoMitigation` wasn't actually ever accessed even from there...)

Reply:
@kirschen @lathomas @ahinzmann (as mmarionncern original developer left the field)

Reply:
perhaps it's a more general XPOG question (as it was initially introduced via XPOG): do we still need to support the muon reco duplicate mitigation?
If not, it could be dropped.